molecule/default/converge.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    - make_cmd: 'make -s install'
+    - aau_gitlab_server_url: ''  # http://your.private-gitlab-server.com  # no trailing slash.
+    - aau_grant_webs: false
+    - aau_grant_secondaries: true
+    - aau_data_source: ''
+
+  pre_tasks:
+    - name: Update apt cache.
+      apt: update_cache=true cache_valid_time=600
+      changed_when: false
+
+    - name: rsync | install rsync
```

Comment:
When the test failis because of 'rsync not found', I believe it's talking about rsync on the ansible controller, not the ansible target. 

So if you duplicate this task but make it a local action, then rsync will exist on both ends: 
```yaml
    - name: install rsync on the ansible controller
      connection: local
      package:
        name: rsync
        state: present
```
I think the controller already runs as root, so you shouldn't need to `become` the task.